### PR TITLE
IDP-443 - Added the topics project

### PR DESCRIPTION
**Pull request by Gcaya**

### CODE REVIEW

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+  </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Workleap.DomainEventPropagation.Abstractions\Workleap.DomainEventPropagation.Abstractions.csproj" PackageVersion="[1.1.1,)"  />
+    </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4">
```
Comment (felpel) [inline src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:20]: You will need to add the `PublicApi.Shipped.txt` and `PublicApi.Unshipped.txt` files manually 🚀

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+  </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Workleap.DomainEventPropagation.Abstractions\Workleap.DomainEventPropagation.Abstractions.csproj" PackageVersion="[1.1.1,)"  />
```
Comment (felpel) [inline src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:16]: Does it work if you omit the `PackageVersion` property?

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+  </PropertyGroup>
+
+    <ItemGroup>
```
Comment (felpel) [inline src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:15]: [Nit] I would reformat the file; the `ItemGroup` node's indent is off by one tab currently.

**Review by felpel — APPROVED** [review]
You will need to add the `PublicApi.Shipped.txt` and `PublicApi.Unshipped.txt` files manually 🚀

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj (hunk 3 above)
Comment (Gcaya) [reply src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:15]: Oh good eye

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @gsoft-inc/tech-platform
```
Comment (felpel) [inline .github/CODEOWNERS:1]: Should it be `@gsoft-inc/internal-developer-platform`?

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj (hunk 1 above)
Comment (Gcaya) [reply src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:20]: ah indeed could add them already

File: src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj (hunk 2 above)
Comment (Gcaya) [reply src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj:16]: Indeed, I'll remove it

File: .github/CODEOWNERS (hunk 4 above)
Comment (Gcaya) [reply .github/CODEOWNERS:1]: ahh men, that copy pasta

**Review by felpel — APPROVED** [review]

**Pull request merged**